src/client/components/Checker.tsx

```diff
@@ -57,7 +57,20 @@ export const Checker: FC<CheckerProps> = ({ config }) => {
     )
   }
 
+  // An input variable is a numeric field if its key name starts with N
+  const isNumericField = (key: string) => {
+    return key[0] === 'N'
```

Comment:
How about looking up for the corresponding `Field` in `fields` and preprocess/cast based on the `field.type`?

Reply:
Makes sense - will modify that in a subsequent PR along with the radio value conversion